llvm/lib/Target/X86/X86.td

```diff
@@ -754,6 +754,10 @@ def TuningUseGLMDivSqrtCosts
 def TuningBranchHint: SubtargetFeature<"branch-hint", "HasBranchHint", "true",
                                         "Target has branch hint feature">;
 
+def TuningAvoidMFENCE
```

Comment:
Should better to use the opposite definition, e.g., `TuningPreferMFENCE` given modern CPUs don't need it?

Reply:
I am sympathetic to that, but also following GCCs lead here.